packages/terra-button/src/terra-dev-site/test/button/ButtonAlignments.test.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import IconCaretUp from 'terra-icon/lib/icon/IconCaretUp';
+import IconCaretDown from 'terra-icon/lib/icon/IconCaretDown';
+import IconEdit from 'terra-icon/lib/icon/IconEdit';
+import IconSuccess from 'terra-icon/lib/icon/IconSuccess';
+import IconTag from 'terra-icon/lib/icon/IconTag';
+import ActionHeader from '../../../../../terra-action-header/src/ActionHeader';
```

Comment:
Is it a good idea to reach into other packages that aren't set as a dependency and import things?

Reply:
This is just a test page created to make changes easier to identify at a glance. This will be removed once we're satisfied with the current changes.

Reply:
Test page nuked! - [2d75edd](https://github.com/cerner/terra-core/pull/2378/commits/2d75edd32aa703ec1bc4f2307ffad27f31f5709f)